utils/src/main/kotlin/Utils.kt

```diff
@@ -151,7 +151,8 @@ fun getCommonFilePrefix(files: Collection<File>) =
         }.reduce { prefix, path ->
             prefix.commonPrefixWith(path)
         }.let {
-            File(it)
+            val commonPrefix = File(it)
+            if (commonPrefix.isDirectory) commonPrefix else commonPrefix.parentFile
```

Comment:
I believe this more generically should be `.exists` as for arguments `/path/to/file` and `/path/to/file` (again) I'd expect `/path/to/file` to be returned. The commit message then needs to be adjusted accordingly.

Reply:
The function is currently only used in SBT which expects it to return a directory. With using `exists()` it could return a normal file, so instead I have renamed the function in another commit to fit the current use case.